src/core/store/modules/lines/lines.reducer.ts

```diff
@@ -215,9 +215,50 @@ const linesReducer = createReducer(linesInitialState, (builder) => {
           categories[category] = [...(categories[category] || []), line.id];
         })
       );
-      state.linesMap = { ...state.linesMap, ...lines };
+
+      const validLines = _.filter(lines, (line) => line.status === 'active' && line.end * 1000 > Date.now());
```

Comment:
@kibagateaux If you review this before I get to it, I'm going to move this code block into a formatter function within `line.ts` called `formatLineCategories` before merging into `develop`.